web/src/app/pages/create-survey/task-details/task-input/option-editor/option-editor.component.html

```diff
@@ -0,0 +1,54 @@
+<!--
+  Copyright 2020 Google LLC
```

Comment:
This should read:

`  Copyright 2023 The Ground Authors.`

Same comment for all new files.